common/persistence/persistence-tests/matchingPersistence_test.go

```diff
@@ -89,14 +91,26 @@ func (s *matchingPersistenceSuite) TestPersistenceStartWorkflow() {
 	startedErr, ok := err1.(*persistence.WorkflowExecutionAlreadyStartedError)
 	s.True(ok)
 	s.Equal(workflowExecution.GetRunId(), startedErr.RunID, startedErr.Msg)
-	s.Equal(persistence.WorkflowStateRunning, startedErr.State, startedErr.Msg)
+
+	// For Cassandra implementation, the conflict row  (startedErr) could be either the concrete or the current execution row
+	// For the concrete row, the state would be WorkflowStateCreated.
+	// For the current row, the state would be WorkflowStateRunning.
+	s.True((startedErr.State == persistence.WorkflowStateRunning) || (startedErr.State == persistence.WorkflowStateCreated),
+startedErr.Msg)
```

Comment:
But the checking order is defined in the code, right? Doe it mean that we always check the current execution row and throw WorkflowStateRunning?